lib/drivers/docs/medtronic600.md

```diff
@@ -0,0 +1,235 @@
+    - `*[ ]` object representing suppressed scheduled basal *for each segment of the basal schedule that the suspension of insulin delivery intersects*
+  - `*[ ]` final (most recent) basal
+    - `*[ ]` basal rate interval with a start time, duration "guessed" from settings, rate delivered, and an annotation re: the "guessed" duration OR
+    - `*[ ]` basal rate interval with a start time and rate, no (= zero) duration
+
+Device-specific? (Add any device-specific notes/additions here.)
+
+#### Boluses
+
+  - `[x]` normal bolus
+    - `[x]` amount of insulin delivered
+    - `[x]` amount of insulin delivery programmed (if differs from actual delivery, in case of bolus interruption, cancellation, etc.)
+  - `[x]` extended bolus
+    - `[x]` amount of insulin delivered
+    - `[x]` duration of insulin delivery
+    - `[x]` amount of insulin delivery programmed (if differs from actual delivery, in case of bolus interruption, cancellation, etc.)
+    - `[x]` duration of insulin delivery programmed (if differs from actual duration, in case of bolus interruption, cancellation, etc.)
+    - `*[ ]` extended bolus that crosses midnight is split into two records
+  - `[x]` combo/dual bolus
+    - `[x]` amount of insulin delivered - immediate (normal)
+    - `[x]` amount of insulin delivered - extended
+    - `[x]` duration of extended insulin delivery
+    - `[x]` amount of immediate insulin delivery programmed (if differs from actual delivery, in case of bolus interruption, cancellation, etc.)
+    - `[x]` amount of extended insulin delivery programmed (if differs from actual delivery, in case of bolus interruption, cancellation, etc.)
+    - `[x]` duration of extended insulin delivery programmed (if differs from actual duration, in case of bolus interruption, cancellation, etc.)
+    - `*[ ]` extended portion of combo bolus that crosses midnight is split into two records
+  - bolus cancellations/interruptions
+    - `*[ ]` represented by a separate event in the device's data log OR
+    - `*[ ]` result in modifications to a bolus event in the device's data log
+  - `[x]` link to "wizard"/calculator entry (via log entry ID or similar)
+
+No Tidepool data model yet:
+
+  - bolus cancellations/interruptions
+    - `[ ]` agent/reason for bolus cancellation
+
+Device-specific? (Add any device-specific notes/additions here.)
+
+#### CBG
+
+(See [the CGM checklist](CGMChecklist.md) instead.)
+
+#### Device Events
+
+  - alarms:
+    - `[x]` low insulin
+    - `[x]` no insulin
+        - `[ ]` needed to infer a suspend (stoppage of all insulin delivery)
+    - `[ ]` low power
+    - `[ ]` no power
+        - `[ ]` needed to infer a suspend (stoppage of all insulin delivery)
+    - `[ ]` occlusion
+        - `[ ]` needed to infer a suspend (stoppage of all insulin delivery)
+    - `[ ]` no delivery
+        - `[ ]` needed to infer a suspend (stoppage of all insulin delivery)
+    - `[ ]` auto-off
```

Comment:
I think Auto-off is now called "auto suspend" on 600-series. Insulin Settings -> Auto Suspend. for "no delivery", block the tubing and/or empty the reservoir. "Over limit" I haven't been able to trigger manually, for "no power" you should ask Eric where to get really poor quality batteries ;)